.travis.yml

```diff
@@ -1,16 +1,19 @@
+dist: xenial
```

Comment:
I'm interested in why we need to be so specific as mention distro?

Reply:
Hi, it is somehow suggested by Travis. They put "info" if there is a fallback on a default setting

Since distros [have different versions of Ruby available](https://rubies.travis-ci.org/) (e.g., bionic drops  Ruby < 2.3), I think that they do this for a safer and more reproducible suite if they change the default.

https://travis-ci.org/github/adzap/timeliness/builds/684909809/config

```
root: missing dist, using the default xenial
root: missing os, using the default linux
```

...BTW, I wanted to switch to bionic, but I forgot 😅

Reply:
Updated distro to the latest LTS available, Focal Fossa

https://docs.travis-ci.com/user/reference/focal/#ruby-support